src/gui/pedalboard/cables/CableView.cpp

```diff
@@ -15,39 +15,54 @@ CableView::CableView (BoardComponent* comp) : board (comp)
 
 CableView::~CableView() = default;
 
-void CableView::paint (Graphics& g)
+bool CableView::mouseOverClickablePort()
 {
-    using namespace CableDrawingHelpers;
-
+    const auto mousePos = Desktop::getMousePosition() - getScreenPosition();
+    nearestPort = portLocationHelper->getNearestPort (mousePos, board);
```

Comment:
I'm having a strange issue where in the Windows VST3 build specifically, this method never recognizes the clickable ports. I think that bug was present before this PR, so I wouldn't worry about trying to fix it now, but I was curious if you'd seen this behaviour at all?

Reply:
Mmm I haven't seen that at all, let me know if there's anything I should test to fix that.

Reply:
Yeah, I'm guessing it's a Windows+VST3 specific thing... I can play around with it later on.